library/IvozProvider/Gearmand/Invoices/Generator.php

```diff
@@ -163,7 +165,14 @@ protected function _getCallData(\IvozProvider\Model\Invoices $invoice)
             );
             $this->_fixedCostTotal  += number_format(ceil($subTotal*10000)/10000, 4);
         }
+
+        $dbAdapter = $invoice->getMapper()->getDbTable()->getAdapter();
+        $updateCallsInvoiceIdQuery = 'UPDATE `kam_acc_cdrs` SET `invoiceId` = ' . $invoice->getId();
+        $updateCallsInvoiceIdQuery .= ' WHERE id in (%s)';
```

Comment:
Is there any difference between using the call ids array or the conditions already existing in `$where` variable?

The performance for these is minimal but I'm imaging myself debugging a query with a list of 5000 ids separeted by comas hehe.